### PR TITLE
fix: update operator capabilities level in CSV

### PR DESCRIPTION
**Pull request by jaideepr97**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

> /kind bug
 /kind chore
> /kind cleanup
> /kind failing-test
> /kind enhancement
> /kind documentation
> /kind code-refactoring

**What does this PR do / why we need it**:

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Review by wtam2018 — COMMENTED** [review]
I think we should leave v0.2.0 manifests and create/fix v0.3.0 manifests.

**Comment by jaideepr97** [comment]
@wtam2018 Looks like the CI is running `make bundle` itself and expects the 0.2.0 csv to also be updated with the annotation

**Comment by jaideepr97** [comment]
@wtam2018 could we merge this?

**Review by wtam2018 — APPROVED** [review]

File: bundle/manifests/argocd-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -124,7 +124,7 @@ metadata:
           }
         }
       ]
-    capabilities: Auto Pilot
+    capabilities: Deep Inisights
```
Comment (wtam2018) [inline bundle/manifests/argocd-operator.clusterserviceversion.yaml:127]: Insights
@jaideepr97  Please fix typo.

File: config/manifests/bases/argocd-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     alm-examples: '[]'
-    capabilities: Auto Pilot
+    capabilities: Deep Inisights
```
Comment (wtam2018) [inline config/manifests/bases/argocd-operator.clusterserviceversion.yaml:6]: Insights
Please fix typo. @jaideepr97

**Pull request merged**